sdk/appconfiguration/azure-appconfiguration/tests/test_azure_appconfiguration_client_async.py

```diff
@@ -1195,6 +1222,22 @@ async def test_monitor_configuration_settings_by_page_etag(self, appconfiguratio
             async for config_setting in config_settings:
                 await client.delete_configuration_setting(key=config_setting.key, label=config_setting.label)
 
+    @app_config_decorator_async
+    @recorded_by_proxy_async
+    async def test_list_snapshot_configuration_settings(self, appconfiguration_connection_string):
```

Comment:
I would expect a test for zero items as well as for some more involved label values (e.g. escaped quotations etc.)